apps/easypid/src/features/receive/slides/CredentialRetrievalSlide.tsx

```diff
@@ -212,7 +218,16 @@ export const CredentialRetrievalSlide = ({
               >
                 {scrollViewHeight && isAllowedToAccept ? (
                   <AnimatedStack key="credential-attributes" entering={FadeIn.duration(200)}>
-                    <CredentialAttributes attributes={attributes} />
+                    {attributes ? (
+                      <CredentialAttributes attributes={attributes} />
+                    ) : (
+                      <Paragraph>
+                        <Trans id="receiveCredential.deferredCredentialParagraph">
+                          Your credentials are not yet ready. We will check once in a while in the background and notify
```

Comment:
Same here, i think we should integrate the interval here, so the user has a sense of when to expect the credential.

Reply:
I think integrating the interval here is a bit more complex than in the other screen. Especially considering we have to deal with three variations of this message (no interval, interval which falls within today, interval which falls outside of today). I think having it in the notification page  should be enough for now.